File: 0191-number-of-1-bits/0191-number-of-1-bits.cs
public class Solution {
    public int HammingWeight(uint n) {
        int result = (int) n;
        string binary = Convert.ToString(result, 2);
        int sum = 0;
        
        for(int i = 0; i < binary.Length; i++){
            if(binary[i] == '1'){
                sum += 1;
            }
        }
        
        return sum;
    }
}

File: 0191-number-of-1-bits/0191-number-of-1-bits.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        #can update int to binary
        # print(bin(n))
        # print('{0:b}'.format(n))
        
        sum = 0
        stringNum = bin(n)
        for o in stringNum:
            if o == "1":
                sum += 1
        
        return sum